src/libCli/OutputFormatting.cpp

```diff
@@ -306,6 +322,34 @@ std::string OutputFormatter::fieldToString(const grpc::protobuf::Message & f_mes
             result += generateHorizontalGuide(nameSize, maxFieldNameSize);
             result += " = " + colorize(ColorClass::MessageTypeName, "{}");
         }
+        if(f_fieldDescriptor->is_map())
+        {
+            //check if the type of the Key-Value simple or not
+            if(is_pair_simple(f_fieldDescriptor->message_type()))
+            {
+                std::map<std::string, std::string>kv_map;
```

Comment:
I like the idea of using map<string,string> but it will not always give correct results.
Example:
10 -> "10"
8 -> "8"

I think we still have to differentiate between different key-types here :(
However, it should be rather easy: As described [here](https://developers.google.com/protocol-buffers/docs/proto3#maps) bytes and floats are not allowed as map keys. This leaves Int, string. and we can probably promote all integers to int64_t/uint64_t

For value types we can stay with string.